magnet_data/magnet_data_client.py

```diff
@@ -14,7 +18,114 @@ def get_pair(base_currency: str, counter_currency: str) -> CurrencyPair:
         )
 
 
+class Holidays(Countries):
+    def __init__(self):
+        self.last_updated = None
+        self.cls = apps.get_model(
+            app_label='magnet_data',
+            model_name='Holiday'
+        )
+
+    def update(self, country_code: str):
+        """
+        Update values stored in the database with what the api returned
+        """
+        now = datetime.datetime.now()
+        threshold = now - datetime.timedelta(1)
+        if self.last_updated is None or self.last_updated < threshold:
+            self.last_updated = now
+            self.cls.update_holidays(country_code=country_code.upper())
+
+    def is_workday(self, date, country_code: str):
+        """
+        Returns True if the given date is not Saturday, Sunday, or
+        Holiday
+        Keyword arguments:
+            from_date -- date to check
+            country-code -- de ISO 3166 country code
+        """
+        self.update(country_code)
+        if date.weekday() == 5:  # Saturday
+            return False
+
+        if date.weekday() == 6:  # Sunday
+            return False
+
+        return not self.cls.objects.filter(
+            date=date,
+            country_code=country_code.upper(),
+        ).exists()
+
+    def get_next_working_day(self,
+                             country_code: str,
+                             working_days: int = 1,
+                             from_date: datetime.date = None,
+                             step: int = 1):
+        """
+        Returns the next date that is a working day.
+        Keyword arguments:
+            country-code -- de ISO 3166 country code
```

Comment:
@munimagnet se pasó un de que supongo era the o que no iba

Reply:
quizás quería poner "the ISO ..."